pack_and_terraform.bash

```diff
@@ -1,10 +1,17 @@
 #!/bin/bash
 # Right now this script guides you through the process
 # of running Packer & Terraform
-
+unamestr=`uname`
+if [[ "$unamestr" == 'Darwin' ]]; then
+    export PATH=${PATH}:/usr/local/opt/gettext/bin
+    if [[ ! -x "$(command -v envsubst)" ]]; then
+        brew install gettext
```

Comment:
Can we not do brew (I don't have it)? Lets either compile from source or just error out if missing and give the user a couple of options.